services/tables/src/main/java/com/linkedin/openhouse/tables/api/validator/impl/PoliciesSpecValidator.java

```diff
@@ -85,6 +89,30 @@ public boolean validate(
                 tableUri);
         return false;
       }
+      if (!validateReplicationPolicy(policies.getReplication())) {
+        failureMessage =
+            String.format(
+                "For table with non-null replication policy, a valid replication schedule %s is mandatory",
+                policies.getReplication());
+        return false;
+      }
+    }
+    return true;
+  }
+
+  private boolean validateReplicationPolicy(Replication replication) {
+    // implement replication config validation
+    if (replication != null) {
+      for (Schedule schedule : replication.getSchedules()) {
+        Map<String, String> config = schedule.getConfig();
+        for (String key : config.keySet()) {
+          try {
+            CronExpression.parse(config.get(key));
```

Comment:
will need to validate cron schedule based on:
1. N days or less schedule (will collect some more data to define N)
2. Schedules less than 1 hour will not be allowed
will do in a separate PR